nums/numpy/api.py

```diff
@@ -27,12 +27,29 @@
 # pylint: disable = redefined-builtin, too-many-lines
 
 
-def _not_implemented(fun):
+def _not_implemented(func):
     # From project JAX: https://github.com/google/jax/blob/master/jax/numpy/lax_numpy.py
     def wrapped(*args, **kwargs):
         # pylint: disable=unused-argument
         msg = "NumPy function {} not yet implemented."
-        raise NotImplementedError(msg.format(fun))
+        raise NotImplementedError(msg.format(func))
+
+    return wrapped
+
+
+# TODO(mwe): Convert this to invoke the NumPy op on a worker instead of the driver
```

Comment:
(nit) can you add a space between TODO and your initials `#TODO (mwe): `, and a period at the end of the sentence?

Reply:
Sure.